shell/renderer/electron_render_frame_observer.cc

```diff
@@ -73,7 +73,17 @@ void ElectronRenderFrameObserver::DidInstallConditionalFeatures(
   // DidCreateScriptContext();
   bool is_main_world = IsMainWorld(world_id);
   bool is_main_frame = render_frame_->IsMainFrame();
-  bool allow_node_in_sub_frames = prefs.node_integration_in_sub_frames;
+
+  // This is necessary because if an iframe is created and a source is not
+  // set, the iframe loads about:blank and creates a script context for the
+  // same. We don't want to create a Node.js environment here because if the src
+  // is later set, the JS necessary to do that triggers illegal access errors
+  // when the initial about:blank Node.js environment is cleaned up. See:
+  // https://source.chromium.org/chromium/chromium/src/+/main:content/renderer/render_frame_impl.h;l=870-892;drc=4b6001440a18740b76a1c63fa2a002cc941db394
+  GURL url = render_frame_->GetWebFrame()->GetDocument().Url();
+  bool allow_node_in_sub_frames =
```

Comment:
Should be checked in `ShouldNotifyClient`, not after it.

Reply:
oops, mis-pasted. fixed!